rpm/expat.spec

```diff
@@ -35,40 +34,41 @@ The expat-devel package contains the libraries, include files and documentation
 to develop XML applications with expat.
 
 %prep
-%setup -q -n expat-%{version}/upstream/expat
+%autosetup -p1 -n %{name}-%{version}/upstream
+
+%build
+pushd expat
 sed -i 's/install-data-hook/do-nothing-please/' lib/Makefile.am
 ./buildconf.sh
 
-%build
 export CFLAGS="$RPM_OPT_FLAGS -fPIC"
 %configure --without-docbook --disable-static
 make %{?_smp_mflags}
 
 %install
+pushd expat
 make install DESTDIR=$RPM_BUILD_ROOT
 
 rm -f $RPM_BUILD_ROOT%{_libdir}/*.la
 
-%check
-make check
```

Comment:
Support dropped by upstream or why this was removed?

Reply:
Just for making it simpler as we don't commonly have these. Can bring it back if you see value there.